test/cluster_commands_on_keys_test.rb

```diff
@@ -98,7 +98,13 @@ def test_touch
       set_some_keys
       assert_equal 1, redis.touch('key1')
       assert_equal 1, redis.touch('key2')
-      assert_equal 1, redis.touch('key1', 'key2')
+      if version < '6.0.0'
+        assert_equal 1, redis.touch('key1', 'key2')
```

Comment:
WI doesn't if fail anymore ? Is it just the hashing that changed?

Reply:
I am not sure. I am somewhat new to redis, and looking at the surrounding tests, I'm surprised that this was passing before.

A search through PRs in antirez/redis shows this change: 
https://github.com/antirez/redis/pull/5391/files

If I am understanding that correctly, the TOUCH command previously did not actually look past the first key argument.

Reply:
That makes sense. Thanks!